src/backend/joanie/core/serializers/client.py

```diff
@@ -580,7 +591,12 @@ def get_organizations(self, instance):
             else:
                 organizations = []
 
-        return OrganizationSerializer(organizations, many=True, read_only=True).data
+        return OrganizationSerializer(
+            organizations,
+            context={"request": self.context.get("request")},
```

Comment:
I'm afraid you can't do that anymore :stuck_out_tongue:

Reply:
I had to add `exclude_abilities` explicitly that is cleaner from my point of view.